practices/coding.md

```diff
@@ -1,41 +1,8 @@
 ---
-title: Base Practices
```

Comment:
I had this a separate file but I see that in this branch you have broken it down into multiple files, which I think it makes sense.